Xamarin.Nuke/Xamarin.Nuke.NET6/Xamarin.Nuke.NET6.csproj

```diff
@@ -0,0 +1,68 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
+    <IsBindingProject>true</IsBindingProject>
+    <NoBindingEmbedding>true</NoBindingEmbedding>
+    <RootNamespace>Xamarin.Nuke</RootNamespace>
+    <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-ios'">14.0</SupportedOSPlatformVersion>
+    <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-maccatalyst'">14.0</SupportedOSPlatformVersion>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <Copyright>Sharpnado 2020</Copyright>
+    <PackageProjectUrl>https://github.com/roubachof/Xamarin.Nuke</PackageProjectUrl>
```

Comment:
The url is in fact https://github.com/roubachof/NukeProxy

Reply:
Fixed this, thanks